### DISCUSSION TITLE
How to download an image

### DISCUSSION THREAD

**Discussion by iPiran** [bareboat-necessities/lysmarine_gen#259] [Q&A] [answered]
Apologies for what is probably a silly question, and I am embarrassed that I haven't found the solution, as I have used many different OS's on my Pi's and built my own 3d printer, but I'm really struggling to get an image.  Cloudsmith just gives me an empty .xz file and the alternate source from GitHub gives me a dodgy part0 file.  I am using a MacBook with the latest OS and unarchiver to try combine the files, any pointers are welcome.

**Comment by mgrouch** [comment, marked as answer]
If you run the whole script it will combine chunk files and create you .xz image file in the directory you run the script from

**Reply by iPiran** [reply]
would I do that via terminal?

**Reply by iPiran** [reply]
Ok that worked thanks
